This patch will update the version number in the release notes.

See http://thread.gmane.org/gmane.comp.security.shorewall/30808

--- shorewall6-lite-4.5.21.7.old/releasenotes.txt	2014-03-08 16:35:39.000000000 +0100
+++ shorewall6-lite-4.5.21.7/releasenotes.txt	2014-03-09 14:38:19.503337459 +0100
@@ -1,5 +1,5 @@
 ----------------------------------------------------------------------------
-	            S H O R E W A L L  4 . 5 . 2 1 . 6
+	            S H O R E W A L L  4 . 5 . 2 1 . 7
                    ------------------------------------
                       M a r c h  0 8 ,  2 0 1 4
 ----------------------------------------------------------------------------
